src/notebooks/main.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import pandas as pd
+from sklearn.metrics import accuracy_score, roc_auc_score
+
+sys.path.append('../preprocess/')
```

Comment:
Kudos for using relative directories (`../`) however the hard-coded file separator `/` will only work on Linux and Macs. Your code will crash on Windows computers, which use a `\` fileseparator.

Instead, use `os.path.join`

Reply:
I agree. Fixed this issue in the new commit for this PR